spec/pluralrules.html

```diff
@@ -21,50 +21,30 @@
           1. Let _options_ be ? ToObject(_options_).
         1. Let _opt_ be a new Record.
         1. Let _s_ be ? GetOption(_options_, *"type"*, *"string"*, « *"cardinal"*, *"ordinal"* », *"cardinal"*).
-        1. Set _pluralRules_.[[type]] to _s_.
+        1. Set _pluralRules_.[[Type]] to _s_.
         1. Let _mnid_ be ? GetNumberOption(_options_, *"minimumIntegerDigits"*, 1, 21, 1).
-        1. Set _pluralRules_.[[minimumIntegerDigits]] to _mnid_.
+        1. Set _pluralRules_.[[MinimumIntegerDigits]] to _mnid_.
         1. Let _mnfd_ be ? GetNumberOption(_options_, *"minimumFractionDigits"*, 0, 20, 0).
-        1. Set _pluralRules_.[[minimumFractionDigits]] to _mnfd_.
+        1. Set _pluralRules_.[[MinimumFractionDigits]] to _mnfd_.
         1. Let _mxfd_ be ? GetNumberOption(_options_, *"maximumFractionDigits"*, _mnfd_, 20, max(_mnfd_, 3)).
-        1. Set _pluralRules_.[[maximumFractionDigits]] to _mxfd_.
+        1. Set _pluralRules_.[[MaximumFractionDigits]] to _mxfd_.
         1. Let _mnsd_ be ? Get(_options_, *"minimumSignificantDigits"*).
         1. Let _mxsd_ be ? Get(_options_, *"maximumSignificantDigits"*).
         1. If _mnsd_ is not *undefined* or _mxsd_ is not *undefined*, then
           1. Let _mnsd_ be ? GetNumberOption(_options_, *"minimumSignificantDigits"*, 1, 21, 1).
           1. Let _mxsd_ be ? GetNumberOption(_options_, *"maximumSignificantDigits"*, _mnsd_, 21, 21).
-          1. Set _pluralRules_.[[minimumSignificantDigits]] to _mnsd_.
-          1. Set _pluralRules_.[[maximumSignificantDigits]] to _mxsd_.
-        1. Let _localeData_ be *%PluralRules%*.[[localeData]].
-        1. Let _r_ be ResolveLocale(*%PluralRules%*.[[availableLocales]], _requestedLocales_, _opt_, *%PluralRules%*.[[relevantExtensionKeys]], _localeData_).
-        1. Set _pluralRules_.[[locale]] to the value of _r_.[[locale]].
-        1. Let _dataLocale_ be _r_.[[dataLocale]].
```

Comment:
no data is used or defined just yet in the spec, we can do that later on if needed, but I don't think it is needed since I'm now referencing to implementation defined pieces.